pkg/roachprod/install/cluster_synced.go

```diff
@@ -263,6 +263,19 @@ func (c *SyncedCluster) GetInternalIP(n Node) (string, error) {
 	return ip, nil
 }
 
+// GetHostname returns the hostname of the specified node.
+func (c *SyncedCluster) GetHostname(n Node) (string, error) {
+	if c.IsLocal() {
+		return c.Host(n), nil
+	}
+
+	ip := c.VMs[n-1].Name
+	if ip == "" {
+		return "", errors.Errorf("no private IP for node %d", n)
```

Comment:
Copy/paste error? It's the hostname we're after.

Reply:
100% a copy/paste error from the function above! Fixed